flask_restalchemy/serialization/modelserializer.py

```diff
@@ -100,8 +98,11 @@ def load(self, serialized, existing_model=None):
             if value is None:
                 deserial_value = value
             elif field.serializer:
-                if isinstance(field.serializer, ModelSerializer) and existing_model:
-                    existing_nested = getattr(existing_model, field_name)
+                if isinstance(field.serializer, ModelSerializer):
+                    if existing_model:
```

Comment:
When posting new recurrent Events, I'm guessing that `existing_model` would be `False` since the Event doesn't have an `id`. Is this correct?

Reply:
No, this can be None but is already handled on line 86